ReactWindows/ReactNative/Bridge/ReactApplicationContext.cs

```diff
@@ -0,0 +1,13 @@
+﻿
+namespace ReactNative.Bridge
+{
+    /// <summary>
+    /// This class serves as a base class on top of <see cref="ReactContext"/>.
+    /// </summary>
+    public class ReactApplicationContext : ReactContext
```

Comment:
ReactContext is a baseline abstraction so that one can extend off the baseline functionality if they waned to.